app/src/test/java/org/openobservatory/ooniprobe/common/ApplicationTest.java

```diff
@@ -0,0 +1,18 @@
+package org.openobservatory.ooniprobe.common;
+
+import androidx.test.filters.SmallTest;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.openobservatory.ooniprobe.RobolectricAbstractTest;
+import org.robolectric.RobolectricTestRunner;
+
+import static org.junit.Assert.assertEquals;
+
+@SmallTest
+public class ApplicationTest extends RobolectricAbstractTest {
+    @Test
+    public void packageName() {
+        assertEquals("org.openobservatory.ooniprobe.dev", a.getPackageName());
```

Comment:
The package name of the app is `org.openobservatory.ooniprobe`
the .dev is the package name of the dev flavour that is only used internally

Reply:
I would always use mainFull for tests as it's the app that lands in the google play store